docker/thingd/config.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
As I said before, you should make this file compatible with the `.conf` format.

Reply:
Done!